truss/remote/baseten/service.py

```diff
@@ -0,0 +1,40 @@
+from typing import Dict
+
+from truss.remote.baseten.auth import AuthService
+from truss.remote.truss_remote import TrussService
+from truss.truss_handle import TrussHandle
+
+
+class BasetenService(TrussService):
+    # TODO(Abu): How to handle liveness/readiness probes from client?
+    def __init__(
+        self,
+        model_id: str,
+        model_version_id: str,
+        is_draft: bool,
+        truss_handle: TrussHandle,
```

Comment:
I wonder if these kwargs also should be stringifiable. Maybe in a followup

Reply:
What's the thinking here? So that a user could create the `TrussService` from the command line?